package.json

```
@@ -23,7 +23,7 @@
         "eslint": "6.x.x",
         "find-rc": "4.x.x",
         "globby": "10.x.x",
-        "handlebars": "4.x.x",
+        "handlebars": "^4.7.3",
```

Does this need to be pinned? If not, this would be better for consistency. 
```suggestion
        "handlebars": "4.x.x",
```